common/lib/dependabot/config/file.rb

```diff
@@ -49,6 +52,27 @@ def self.parse(config)
 
         File.new(updates: parsed[:updates], registries: parsed[:registries])
       end
+
+      private
+
+      def ignore_conditions(cfg)
+        ignores = cfg&.dig(:ignore) || []
+        ignores.map do |ic|
+          Dependabot::Config::UpdateConfig::IgnoreCondition.new(
+            name: ic[:"dependency-name"],
```

Comment:
Nit but wondering if `dependency_name` would be clearer once we add other things like update-types, dependency-types
```suggestion
            dependency_name: ic[:"dependency-name"],
```